### DISCUSSION TITLE
Bug: 更新后左边导航栏的 设置+依赖管理 图标消失，插件市场永远加载中

### DISCUSSION THREAD

**Discussion by eric2788** [koishijs/koishi#1224] [Q&A] [answered]
![image](https://github.com/koishijs/webui/assets/6276840/07e5844a-9a73-4036-a893-36fb819ca4b1)
![image](https://github.com/koishijs/webui/assets/6276840/5e7488ca-444a-4082-a59f-b3bbb1a8594b)

console 没有显示错误，只有如上的 warnings
尝试过重启koishi依然不行

配置如下:

System:
    OS: Linux 5.10 Alpine Linux
    CPU: (4) arm64 Cortex-A72

Binaries:
    Node: 18.17.1
    Yarn: 3.6.2

Koishi:
    Core: 4.15.0
    Console: 5.16.1

我现在除了没法使用插件市场外，因为插件配置以及依赖管理消失的原因也没办法删除/新增插件

**Comment by ilharp** [comment]
推荐使用问题前往 Discussions 版或 Koishi 论坛讨论。已将此 Issue 转换为 Discussion。

这个问题并非 Koishi 的 bug，而是你遇到了名为「多例」的问题。该问题在 Koishi 论坛上已有多个讨论贴及解决方案，可以参考。

<https://k.ilharp.cc/5473>

**Reply by eric2788** [reply]
我尝试跟了链接里面的教学

![image](https://github.com/koishijs/koishi/assets/6276840/e297a79f-89de-4da4-b2bd-c7dd003a68b2)

我发现我好像没有这个功能？我右键之后并没有出现高级这个选项

**Reply by ilharp** [reply, marked as answer]
通知区域图标由 Koishi Desktop 提供，而你似乎并不在使用 Koishi Desktop。你需要在你所在的环境内完成教程内的操作，如编辑文件和使用终端。

如果你使用 Docker 环境，则可以直接使用 Docker 的「Exec」功能附加到容器，工作目录内就有你需要编辑的文件，你也可以直接运行终端命令。

**Reply by eric2788** [reply]
了解！感谢解答
